NOTICE

```diff
@@ -3,3 +3,6 @@ Copyright 2017 The Apache Software Foundation.
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+This product includes software from the
```

Comment:
Ok good, we only need to declare the ANTLR bundling in the binary distribution then.